src/index.html

```diff
@@ -1,13 +1,426 @@
+            >
+              Marina Palms
+            </a>
+
+              / North Miami Beach, FL 33162
+            <div class="place__description">
+
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-6">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/3.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--desktop-7-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/4.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <a
+              href="https://goo.gl/maps/FxNXpMdnRnhbJq9c6"
+              class="place__link"
+            >
+              Edition Residens
+            </a>
+
+            / Miami Beach, FL 33139
+            <div class="place__description">
+
+            </div>
+          </div>
+
+          <div class="gallery__place place">
+            <div class="place__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/5.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+
+                <div class="grid__item grid__item--desktop-5-12">
+                  <div class="place__photo-container">
+                    <img
+                      src="images/photos/6.jpg"
+                      alt="Place 1"
+                      class="place__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <a
+              href="https://goo.gl/maps/FxNXpMdnRnhbJq9c6"
+              class="place__link"
+            >
+              Faena House
+            </a>
+
+            / Miami Beach, FL 33140
+            <div class="place__description">
+
+            </div>
+          </div>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--6-7">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section testimonials" id="testimonials">
+        <div class="container">
+          <h2 class="page__section-title">
+            Testimonials
+          </h2>
```

Comment:
These pictures should also increase on hover
![image](https://user-images.githubusercontent.com/37161315/170516511-83528e4e-e6bb-4ccf-9f33-a8fb9d6acacb.png)

Reply:
done